src/jsonld/corporateContact.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Head from 'next/head';
+
+import markup from '../utils/markup';
+
+const buildContactPoints = contactPoints =>
+  contactPoints.map(
+    contacts => `{
+    "@type": "ContactPoint",
+    "telephone": "${contacts.telephone}",
+    "contactType": "${contacts.contactType}"
+    }`,
+  );
+
+const CorporateContactJsonLd = ({ url, logo, contactPoints = [] }) => {
+  const jslonld = `{
+    "@context": "https://schema.org",
+    "@type": "Organization",
+    "url": "${url}",
+    "logo": "${logo}",
```

Comment:
As `logo` is not required, this can be empty which would cause invalid JSON.

You can see examples of how this is handled in some of the other jsonld files.

Reply:
updated, thanks